lab-andrew/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+
+
+gulp.task('test', function() {
+  return gulp.src(__dirname + '/test/*.js')
+  .pipe(mocha());
+});
+
+gulp.task('lint', function() {
+  return gulp.src(['**/*.js', '!node_modules/**'])
+  .pipe(eslint()).pipe(eslint.format());
+});
+
+gulp.task('watcher', function() {
+  gulp.watch(['**/*.js','!node_modules**'], ['test', 'lint']);
+});
+
+gulp.task('default', ['test', 'lint', 'watcher'], function() {
```

Comment:
You can remove the anonymous function at the end of this task like so:
gulp.task('default', ['test', 'lint', 'watcher']);